packages/vuetify/src/components/VOtpInput/VOtpInput.tsx

```diff
@@ -212,6 +212,8 @@ export const VOtpInput = genericComponent<VOtpInputSlots>()({
     useRender(() => {
       const [rootAttrs, inputAttrs] = filterInputAttrs(attrs)
 
+      const { loader: _, ...vFieldSlots } = slots
```

Comment:
Use omit? https://github.com/vuetifyjs/vuetify/blob/8030d407c5767f96c0474d94828b2a79b40e7c17/packages/vuetify/src/util/helpers.ts#L257-L266

Reply:
`prop: _` is also being pervasively used. Was there any obvious benefit of using omit? I guess both are okay so just need to pick one of them?

https://github.com/vuetifyjs/vuetify/blob/d0765f1125b5f3555acbbcf05aeec3bfeed6de72/packages/vuetify/src/components/VTextField/VTextField.tsx#L160